src/MizarWidgets/MizarMainWindow.ui

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>MainWindow</class>
+ <widget class="QMainWindow" name="MainWindow">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>986</width>
+    <height>449</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Mizar</string>
+  </property>
+  <widget class="QWidget" name="central_widget_">
+   <layout class="QVBoxLayout" name="verticalLayout">
+    <item>
+     <widget class="QTabWidget" name="tab_widget_">
+      <property name="currentIndex">
+       <number>0</number>
+      </property>
+      <widget class="QWidget" name="sampling_with_frametrack_tab_">
+       <attribute name="title">
+        <string>Sampling with Frame Track</string>
+       </attribute>
+       <layout class="QVBoxLayout" name="verticalLayout_2">
+        <item>
+         <layout class="QHBoxLayout" name="layout">
```

Comment:
sampling_with_frame_track_h_box_layout_

Reply:
But it's also vertical layout. Just like the one you suggest to call sampling_with_frame_track_vertical_layout_

Reply:
... NVM, I've just removed the "layout" and the item it was wrapped in.